internal/clusterfeature/features/logging/config.go

```diff
@@ -29,7 +29,13 @@ func (c Config) Validate() error {
 		return errors.New("logging namespace is required")
 	}
 
-	// TODO: validate chart config: image and tag are not empty!
+	if c.Charts.Operator.Chart == "" {
```

Comment:
Why not implement a `Validate` method on `ChartConfig` the same way you did for the monitoring config?